djchoices/tests/test_choices.py

```diff
@@ -125,6 +125,16 @@ def test_numeric_class_validator(self):
         self.assertRaises(ValidationError, NumericTestClass.validator, 6)
         self.assertRaises(ValidationError, NumericTestClass.validator, 7)
 
+    def test_validation_error_message(self):
+        from django.core.exceptions import ValidationError
+
+        message = ("Select a valid choice. 4 is not "
+                   "one of the available choices.")
+
+        with self.assertRaises(ValidationError) as context:
```

Comment:
you can also use `with self.assertRaisesMessage(ValidationError, 'Select a valid choice. 4 is not one of the available choices'):`